modules/openapi-generator/src/main/resources/rust/Cargo.mustache

```diff
@@ -78,3 +78,22 @@ google-cloud-token = "^0.1"
 {{/supportTokenSource}}
 {{/supportAsync}}
 {{/reqwest}}
+{{#reqwestTrait}}
+async-trait = "^0.1"
+reqwest = { version = "^0.12", features = ["json", "multipart"] }
+{{#supportMiddleware}}
+reqwest-middleware = { version = "^0.3", features = ["json", "multipart"] }
+{{/supportMiddleware}}
+{{#supportTokenSource}}
+# TODO: propose to Yoshidan to externalize this as non google related crate, so that it can easily be extended for other cloud providers.
+google-cloud-token = "^0.1"
+{{/supportTokenSource}}
+{{#mockall}}
+mockall = { version = "^0.13", optional = true}
+{{/mockall}}
+{{/reqwestTrait}}
+
+[features]
+{{#mockall}}
```

Comment:
what about putting `[features]` inside the `mockall` mustache tags?

Reply:
@wing328 I thought about that originally, but figured it would be easier to always add `[features]` so added new features in the future would be easier.

ie.
```toml
[features]
{{#mockall}}
mockall = ["dep:mockall"]
{{/mockall}}
{{#foo}}
foo = ["dep:foo"]
{{/foo}}
{{#bar}}
bar = ["dep:bar"]
{{/bar}}
```

But I do not have a strong opinion on this, we can change if there is a reason to :+1:

Reply:
that's ok. i may file a PR later to update it